webapp/sources/scala-ldap/README.asciidoc

```diff
@@ -3,7 +3,7 @@ Scala-LDAP: an LDAP client library for Scala
 
 This project is part of Rudder - configuration management made easy.
 
-See: http://rudder-project.org for more information.
+See: https://www.rudder.io/ for more information.
 
 [horizontal]
 Bug report:: http://www.rudder-project.org/redmine/projects/scala-ldap/issues
```

Comment:
The URLs are also outdated

Reply:
FYI for rudder-techniques I just redirected to rudder's main README (https://github.com/Normation/rudder-techniques/blob/master/README.adoc) to avoid having to maintain general information in all READMES

Reply:
(Authors section points to normation.com, and seems to miss some "new" developers.

Reply:
done